CMakeLists.txt

```diff
@@ -312,6 +312,7 @@ if(WITH_NETWORK_BACKEND)
 		list(APPEND LIBS_TO_LINK ${AVAHI_LIBRARIES})
 	elseif(WIN32)
 		set(HAVE_DNS_SD ON)
+		set(CMAKE_C_FLAGS "-D_WIN32_WINNT=0x0600 ${CMAKE_C_FLAGS}")
```

Comment:
I wouldn't want this flag set when building on Windows (not cross compiling) since it freezes the value of D_WIN32_WINNT to 0x0600. On windows this flag gets set automatically. However you could check if the project is being cross-compiled and then set this flag (check if CMAKE_COMPILER_IS_GNUCC or Clang).